src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1389,18 +1389,14 @@
      * class object.
      *
      * <p>For example:</p>
-     *
-     * <pre>
-     * int.class
+     * <pre> int.class
      * </pre>
      * <p>parses as:</p>
-     * <pre>
-     * +--EXPR
-     *     |
-     *     +--DOT (.)
-     *         |
-     *         +--LITERAL_INT (int)
-     *         +--LITERAL_CLASS (class)
+     * <pre> 
+     * `--EXPR -&gt; EXPR
+     *     `--DOT -&gt; .
+     *         |--LITERAL_INT -&gt; int
+     *         `--LITERAL_CLASS -&gt; class
```

Comment:
Sorry for confusion. Please also remove `-- in first line and adjust indentation accordingly in others.

Reply:
commit 3fef3f2 has made the required changes.